content/es/_index.md

```diff
@@ -35,7 +35,7 @@ Para una lista de contribuidores completa, consulte [esta página de GitHub](htt
 
 Esta versión del glosario Cloud Native en español incluye contribuciones realizadas por
 [Victor Morales](https://www.linkedin.com/in/electrocucaracha/),
-[Rael Garcia Arnes](https://www.linkedin.com/in/rael/),
+[Mariano Gabriel González](https://www.linkedin.com/in/marianogg9/),
```

Comment:
minor question to the team.

I hope to check the name needs to be replaced (agreed in the team?)
I think this section is to acknowledge major contributors so does not need to be aligned with the reviewers/approvers. (the team can add many more if the team think they are major contributors)

Reply:
Thanks @seokho-son for the comments, I will double check with the team, I think it was an agreement with the team too. cc: @electrocucaracha @ramrodo

Reply:
Yes. We agree with the change. We're not aligned with the reviewers/approvers, just adding a major contributor and replacing an inactive contributor for now.

Reply:
Well, @raelga was crucial for the team, but he has taken some personal time lately. In the other hand, @marianogg9 has been doing an amazing work during this year. BTW, this is a life document and we can adjust it later.